omnibus/config/patches/snowflake-connector-python-py2/dependencies.patch

```diff
@@ -1,26 +1,44 @@
 diff --git a/setup.py b/setup.py
-index 0da6a8a..509e3a1 100644
+index 0da6a8a..bbf3481 100644
 --- a/setup.py
 +++ b/setup.py
-@@ -182,16 +182,16 @@ setup(
-         'botocore>=1.5.0,<1.14.0',
-         'requests<2.23.0',
-         'urllib3>=1.20,<1.26.0',
+@@ -178,25 +178,25 @@ setup(
+     install_requires=[
+         'azure-common<2.0.0',
+         'azure-storage-blob<12.0.0',
+-        'boto3>=1.4.4,<1.11.0',
+-        'botocore>=1.5.0,<1.14.0',
+-        'requests<2.23.0',
+-        'urllib3>=1.20,<1.26.0',
 -        'certifi<2021.0.0',
++        'boto3',
++        'botocore',
++        'requests',
++        'urllib3',
 +        'certifi',
```

Comment:
Would it make sense to keep the major version constraint (maybe not for `certifi` but for the rest)

Reply:
I think not because they appear to have been defined arbitrarily, and our E2E tests in CI will catch any issues

Reply:
LGTM then